<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapDownloader</class>
 <widget class="QMainWindow" name="MapDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QOfflineMapServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Map downloader</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Enter URL of the tile map:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MapURL">
          <property name="text">
           <string>http://opencache2.statkart.no/gatekeeper/gk/gk.open_gmaps?layers=topo4&amp;zoom=${z}&amp;x=${x}&amp;y=${y}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Zoom range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Zoom">
            <property name="text">
             <string>1-17</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Select bounding box of the downloaded map:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Latitude    </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>1.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>2.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>2.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="Longitude1">
            <property name="text">
             <string>6.5E</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>1.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="Longitude2">
            <property name="text">
             <string>6.6E</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Longitude </string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="Latitude1">
            <property name="text">
             <string>62.3N</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="Latitude2">
            <property name="text">
             <string>62.4N</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="ResetPoints">
          <property name="text">
           <string>Reset points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Output directory where the folder with the map tiles will be created:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditOutDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ChangeDirectory">
            <property name="text">
             <string>Change directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Download">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Download progress:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Offline server</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Source folder with the map tiles:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditInDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SelectFolder">
            <property name="text">
             <string>Select a folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Server port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="serverPortSpinBox">
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>8080</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LocalHost">
          <property name="text">
           <string>Run server</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Running at:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStatus">
            <property name="text">
             <string>not started</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Log console:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>MapURL</tabstop>
  <tabstop>Zoom</tabstop>
  <tabstop>Latitude1</tabstop>
  <tabstop>Latitude2</tabstop>
  <tabstop>Longitude1</tabstop>
  <tabstop>Longitude2</tabstop>
  <tabstop>ResetPoints</tabstop>
  <tabstop>lineEditOutDir</tabstop>
  <tabstop>ChangeDirectory</tabstop>
  <tabstop>Download</tabstop>
  <tabstop>lineEditInDir</tabstop>
  <tabstop>SelectFolder</tabstop>
  <tabstop>serverPortSpinBox</tabstop>
  <tabstop>LocalHost</tabstop>
  <tabstop>lineEditStatus</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
